File: C questions/Q8.c


#include <stdio.h>

int main() {
    int arr[] = {1, 2, 3,4};  
    int sum = 0, count = 0; 
    int len = sizeof(arr) / sizeof(arr[0]); 
    for (int i = 0; i < len; i++) {
        sum += arr[i];  
        count++; 
    }
    float average = (float)sum / count;
    printf("The average is: %.2f\n", average);
    return 0;  
}

File: javascript 90 questions/Q8.js

function calculateAverage(arr) {
    let sum = 0; 
    let count = 0; 
    for (let i = 0; i < arr.length; i++) {
        sum += arr[i]; 
        count++; 
    }
    let average = sum / count; 
    return average; 
}
const numbers = [1,2,3,4];
const avg = calculateAverage(numbers);
console.log("Average:", avg); 
